tests/python/test_policy_service.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+from unittest.mock import patch, MagicMock
+import json
+from policyengine_api.services.policy_service import PolicyService
+
+
+@pytest.fixture
+def mock_database():
+    with patch("policyengine_api.services.policy_service.database") as mock_db:
+        yield mock_db
+
+
+@pytest.fixture
+def sample_policy_data():
+    return {
+        "id": 1,
+        "country_id": "US",
+        "policy_json": json.dumps({"param": "value"}),
+        "policy_hash": "hash123",
+        "label": "test_policy",
+        "api_version": "1.0.0",
+    }
+
+
+@pytest.fixture
+def policy_service():
+    return PolicyService()
+
+
+class TestPolicyService:
+
+    test_policy_id = 8  # Arbitrary value higher than 5
+
+    def test_get_policy_success(
+        self, policy_service, mock_database, sample_policy_data
+    ):
+        # Setup mock
+        mock_database.query.return_value.fetchone.return_value = (
+            sample_policy_data
+        )
+
+        # Test
+        result = policy_service.get_policy("us", self.test_policy_id)
+
+        print(result)
+        print(result["policy_json"])
+        print(type(result["policy_json"]))
+        # Verify
+        assert result is not None
+        assert isinstance(result["policy_json"], dict)
+        assert result["policy_json"]["param"] == "value"
```

Comment:
**suggestion, non-blocking** I think you can do all this in one go with assertpy's assert_that? (not something I've used a ton). see [dict-comparison](https://github.com/assertpy/assertpy?tab=readme-ov-file#dict-comparison)